profile/src/main/java/org/apache/karaf/profile/assembly/Builder.java

```diff
@@ -1023,6 +1023,11 @@ private Profile startupStage(Profile startupProfile) throws Exception {
         for (Map.Entry<Integer, Set<String>> entry : invertedStartupBundles.entrySet()) {
             String startLevel = Integer.toString(entry.getKey());
             for (String location : new TreeSet<>(entry.getValue())) {
+                // remove any custom deployer prefix
+                int mvnIndex = location.indexOf("mvn:");
```

Comment:
Same question as in the other PR: I don't understand why you remove the `mvn` prefix from the bundle location.

Reply:
Like i said on the other PR, `mvn:` is not being removed.